utils.js

```diff
@@ -21,6 +21,21 @@ let chakram = require('chakram');
  */
 let utils = {
     /**
+     * Ensure that a Chakram response object is sane before using it.
+     * Throws if the response is unreasonable.
+     *
+     * @param {ChakramResponse} response Object to check.
+     */
+    assertSaneResponse: function assertSaneResponse(response) {
```

Comment:
Would we want to also assertSaneResponse for _oauthRequest as well? That doesn't necessarily have to be done in this PR, but it's just something to think about.

Reply:
Yes, I should definitely do that!